bin/use_simulator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+set -o pipefail
+
+project="$1"
+scheme="$2"
+
+xcodebuild -project "$project" -scheme "$scheme" -showBuildSettings \
+  | grep CORRESPONDING_SIMULATOR_SDK_NAME \
+  2>&1 > /dev/null
```

Comment:
As in: redirecting `stderr` is redundant here since the pipe will fail? I'm not sure how this should be written, honestly.